enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/søknad/UnderUtdanning.kt

```diff
@@ -6,5 +6,9 @@ data class UnderUtdanning(val skoleUtdanningssted: Søknadsfelt<String>,
                           val heltidEllerDeltid: Søknadsfelt<String>,
                           val hvorMyeSkalDuStudere: Søknadsfelt<Int>?,
                           val hvaErMåletMedUtdanningen: Søknadsfelt<String>?,
-                          val utdanningEtterGrunnskolen: Søknadsfelt<Boolean>,
+                          val utdanningEtterGrunnskolen: Søknadsfelt<Boolean>?, //arbeidsmengde
```

Comment:
Skal ikke dette spørsmåleet stilles? Hva menes med //arbeidsmengde

Reply:
Denne fila påvirkes ikke av barnetilsyn ettersom barnetilsyn kun gjelder for de som er i jobb, ikke under utdanning

Reply:
Det ble helt feil, revertert.